edx_proctoring/templates/proctoring/seq_proctored_exam_rejected.html

```diff
@@ -0,0 +1,55 @@
+{% load i18n %}
+<div class="failure sequence proctored-exam" data-exam-id="{{exam_id}}">
+  <h3>
+    {% blocktrans %}
+      Your proctoring session was reviewed and did not pass requirements
+    {% endblocktrans %}
+  </h3>
+
+  <h4>
+    {% blocktrans %}
+      Your Proctoring Session review: <b class="failure"> Failed </b>
+    {% endblocktrans %}
+  </h4>
+
+  <p>
+    {% blocktrans %}
+      Your session was reviewed by one of our proctors and some activities noted violate the
+      <a href="#">requirements and guidelines</a> needed for a successful review,
+    {% endblocktrans %}
```

Comment:
You are not eligible to purchase academic credit for this course, regardless of your final grade in the course. If you want to appeal this result, contact edX Support.